corehq/apps/cloudcare/static/cloudcare/js/backbone/apps.js

```diff
@@ -677,6 +678,28 @@ cloudCare.AppView = Backbone.View.extend({
         data.onload = function (adapter, resp) {
             cloudCare.dispatch.trigger("form:ready", form, caseModel);
         };
+        data.answerCallback = function(sessionId) {
```

Comment:
It looks like the feature flag just hides the divs but still makes all these ajax calls. Given what we've learned recently about cloudcare performance I'm not comfortable with that. I think you need to initialize this with the feature enabled so that you can do nothing on the answercallback if the flag isn't enabled.

Also, I think we may want to make this a button rather than a magic thing eventually.